axum-extra/CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning].
   introduced as an implicit feature through an optional dependency which was no
   longer being used ([#3298])
 - **breaking:** `option_layer` now maps the `Response` body type to `axum::body::Body` ([#3469])
+- **breaking:** `axum` is an optional dependency.
```

Comment:
For user-facing changelog, I think it would be better to just say we have added new features which may need to be explicitly opted in. The list following this already just lists them.

The fact that axum became an optional dependency, while being the driving reason behind this change, is more of a result of the new features.

Reply:
Updated, and added a link to this pull request.